Gemfile.lock

```diff
@@ -502,4 +478,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   2.3.18
+   2.3.21
```

Comment:
how about keeping the default bundler version 2.3.7 for now? to avoid the ruby-cleanup error.